### DISCUSSION TITLE
Have to reload .ideavimrc to use easy motion

### DISCUSSION THREAD

**Discussion by Wkjlovec** [JetBrains/ideavim#924] [Q&A] [answered]
I have installed IdeaVim-EasyMotion and AceJump plugins,and add " Plug 'vim-easymotion' " to my .ideavimrc,
but every time when  restart idea,i have to reload .ideavimrc (like :source ~/.ideavimrc) to use easy motion features.
My IdeaVim version :2.12.0

**Comment by lippfi** [comment]
Sorry for the late response
Could you please attach your .ideavimrc?

**Reply by Wkjlovec** [reply]
Glad to hear from you(me late too)
Sorry I had to convert the file format to txt to ensure transmission
[ideavimrc.txt](https://github.com/user-attachments/files/16761933/ideavimrc.txt)

**Reply by lippfi** [reply, marked as answer]
It might be caused by mapleader defined after the easymotion mappings
Could you please move the `let mapleader = "\<space>"`  line up and tell if the issue is gone?

**Reply by Wkjlovec** [reply]
That worked perfectly. Thanks for your help!
